.env.development

```diff
@@ -0,0 +1,7 @@
+REACT_APP_API_KEY = AIzaSyAZYpkA5loU8c-sCTs-MAX72Wxzpd5boiE
```

Comment:
Instead of checking in the `.env` file, I would recommend updating the vercel app so that the sensitive information remains hidden. @atapas your input is required.

Reply:
not sure about this i kept it as it is.

Reply:
I think we need both. For the local dev env you need .env file and it shouldn't be committed

For the production, we need to add those to the hosting server. You can DM me the details, I will add them.

Reply:
Yes, please have this file locally to you however do not check-in. 
Name the file as `.env.development.local` and .gitignore will do the rest.

Reply:
Ok thanks @koustov 

@Angryman18 lets get in a call today to set up the firbase config using ReactPlay cred and set in prod env var. We can test and accept the PR after that.

Reply:
Remove this and do not commit keys